### PR TITLE
LRQA-75272 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-75272

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/userbar/UserBar.path [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <tbody>
 <tr>
 	<td>USER_AVATAR_DROPDOWN_ITEM</td>
-	<td>//div[contains(@class,'dropdown-menu-personal-menu') and contains(@class,'show')]//a[contains(.,'${key_dropdownItem}')]</td>
+	<td>//div[contains(@class,'dropdown-menu-personal-menu') and contains(@class,'show')]//a[contains(.,'${key_dropdownItem}')] | //div[contains(@class,'dropdown-menu-personal-menu') and contains(@class,'show')]//button[contains(.,'${key_dropdownItem}')]</td>
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/userbar/UserBar.path]: Please try to combine these 2 locator, thanks.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @melodywu ,
The 2 locator can be combined by using *, please try t combine them to see if it works for other tests, thanks.

**Comment by melodywu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

**Comment by Sharryxue** [comment]
No changes needed. Rebase and resend.
https://github.com/Sharryxue/liferay-portal/pull/416

**Pull request closed without merging**